src/test_utils.ts

```diff
@@ -19,15 +19,23 @@ import {
   RenderResult,
   act,
   render,
+  waitForDomChange,
+  waitForElement,
+  waitForElementToBeRemoved,
 } from '@testing-library/react';
 
 export function delay(timeoutMs: number): Promise<void> {
   return new Promise(resolve => setTimeout(resolve, timeoutMs));
 }
 
-// Render a component and wait for some async DOM changes on init.
-// This can be used to silence warnings of changing the DOM outside
-// act(...), e.g. for RMWC <Tab>s or <Dialog>s.
+/**
```

Comment:
Suggestion on name: `renderWithAsyncEffects(<div />)`

Reply:
Removed this function. See discussion above.